dev/io.openliberty.restfulWS.mpMetrics.filter/src/io/openliberty/restfulws/mpmetrics/MonitorAppStateListener.java

```diff
@@ -39,20 +31,6 @@ public class MonitorAppStateListener implements ApplicationStateListener {
 
     static MonitorMetricsHandler monitorMetricsHandler;
 
-    private final static String MONITORING_GROUP_FILTER = "filter";
```

Comment:
This is related to disabling the filter if the "REST" string is not part of the `filter` attribute for the `monitor` element in serer.xml. But since we are removing the filter and using a callback mechanism instead, its no longer needed here.